### PR TITLE
Added the Emergency Notification methods

### PR DESCRIPTION
**Pull request by jchavez443**

### CODE REVIEW

File: lib/aeuis.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+var Client = require("./client");
+var AEUIS_PATH = "aeuis";
+
+function Aeuis(){
```
Comment (izakfilmalter) [inline lib/aeuis.js:4]: Does this need to be a function? Can it just be an object?

File: lib/aeuis.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    });
+};
+
+Aeuis.get = function(client, acid, callback){
+    if(arguments.length === 2){
+        callback = acid;
+        acid = client;
+        client = new Client();
+    }
+    client.makeRequest("get", client.concatAccountPath(AEUIS_PATH) + `/${acid}`, function(err, res){
```
Comment (izakfilmalter) [inline lib/aeuis.js:29]: Might as well do
```
`${client.concatAccountPath(AEUIS_PATH)}/${acid}`
```

File: lib/emergencyNotification.js [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+    });
+};
+
+EmergencyNotification.getRecipient = function(client, enrid, callback){
+    if(arguments.length === 2){
+        callback = enrid;
+        enrid = client;
+        client = new Client();
+    }
+    client.makeRequest("get", client.concatAccountPath(EMERGENCY_NOTIFICATION_RECIPIENTS) + `/${enrid}`,  function(err, res){
```
Comment (izakfilmalter) [inline lib/emergencyNotification.js:33]: Might as well do
```
`${client.concatAccountPath(EMERGENCY_NOTIFICATION_RECIPIENTS)}/${enrid}`
```

File: lib/emergencyNotification.js [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+};
+
+EmergencyNotification.prototype.replaceRecipient = function(client, recipient, callback){
+    if(arguments.length === 2){
+        callback = recipient;
+        recipient = client;
+        client = new Client();
+    }
+
+    client.makeRequest("put", client.concatAccountPath(EMERGENCY_NOTIFICATION_RECIPIENTS) + `/${this.enrid}`, recipient, function(err, res){
```
Comment (izakfilmalter) [inline lib/emergencyNotification.js:66]: Might as well do
```
${client.concatAccountPath(EMERGENCY_NOTIFICATION_RECIPIENTS)}/${this.enrid}`
```

File: lib/emergencyNotification.js [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+    });
+};
+
+EmergencyNotification.prototype.deleteRecipient = function(client, callback){
+    if(arguments.length === 1){
+        callback = client;
+        client = new Client();
+    }
+
+    client.makeRequest("put", client.concatAccountPath(EMERGENCY_NOTIFICATION_RECIPIENTS) + `/${this.enrid}`, function(err, res){
```
Comment (izakfilmalter) [inline lib/emergencyNotification.js:82]: Might as well do
```
${client.concatAccountPath(EMERGENCY_NOTIFICATION_RECIPIENTS)}/${this.enrid}`
```

File: lib/emergencyNotification.js [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+    });
+};
+
+EmergencyNotification.getGroupOrder = function(client, orderId, callback){
+    if(arguments.length === 2){
+        callback = orderId;
+        orderId = client;
+        client = new Client();
+    }
+    client.makeRequest("get", client.concatAccountPath(EMERGENCY_NOTIFICATION_GROUP_ORDERS) + `/${orderId}`, function(err, res){
```
Comment (izakfilmalter) [inline lib/emergencyNotification.js:114]: Might as well do
```
${client.concatAccountPath(EMERGENCY_NOTIFICATION_GROUP_ORDERS)}/${orderId}`
```

File: lib/emergencyNotification.js [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+    });
+};
+
+EmergencyNotification.getGroup = function(client, engid, callback){
+    if(arguments.length === 2){
+        callback = engid;
+        engid = client;
+        client = new Client();
+    }
+    client.makeRequest("get", client.concatAccountPath(EMERGENCY_NOTIFICATION_GROUPS) + `/${engid}`, function(err, res){
```
Comment (izakfilmalter) [inline lib/emergencyNotification.js:146]: Might as well do
```
${client.concatAccountPath(EMERGENCY_NOTIFICATION_GROUPS)}/${enrid}`
```

File: lib/emergencyNotification.js [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+    });
+};
+
+EmergencyNotification.getEndpointOrder = function(client, engId, callback){
+    if(arguments.length === 2){
+        callback = engId;
+        engId = client;
+        client = new Client();
+    }
+    client.makeRequest("get", client.concatAccountPath(EMERGENCY_NOTIFICATION_ENDPOINT_ORDERS) + `/${engId}`, function(err, res){
```
Comment (izakfilmalter) [inline lib/emergencyNotification.js:194]: Might as well do
```
${client.concatAccountPath(EMERGENCY_NOTIFICATION_ENDPOINT_ORDERS)}/${enrid}`
```

File: lib/emergencyNotification.js [hunk 9]
```diff
@@ -0,0 +1,222 @@
+var Client = require("./client");
+var EMERGENCY_NOTIFICATION_RECIPIENTS = "emergencyNotificationRecipients";
+var EMERGENCY_NOTIFICATION_GROUP_ORDERS = "emergencyNotificationGroupOrders";
+var EMERGENCY_NOTIFICATION_GROUPS = "emergencyNotificationGroups";
+var EMERGENCY_NOTIFICATION_ENDPOINT_ORDERS = "emergencyNotificationEndpointOrders";
+
+function EmergencyNotification(){
```
Comment (izakfilmalter) [inline lib/emergencyNotification.js:7]: Again probably could be an object, rather than a function.

Comment (jchavez443) [reply lib/emergencyNotification.js:7]: This was done to keep in the practice of how the SDK was written so there no varying styles.  But if the SDK is every refactored a class would be a better way.

File: lib/aeuis.js (hunk 1 above)
Comment (jchavez443) [reply lib/aeuis.js:4]: This was done to keep in the practice of how the SDK was written so there no varying styles. But if the SDK is every refactored a class would be a better way.

Comment (jchavez443) [reply lib/aeuis.js:4]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes

File: lib/aeuis.js (hunk 2 above)
Comment (jchavez443) [reply lib/aeuis.js:29]: Again the template string would be better but just to keep the code looking similar we went with concatenation.  These are all great points!

This SDK was written a while ago.

File: lib/emergencyNotification.js (hunk 3 above)
Comment (jchavez443) [reply lib/emergencyNotification.js:33]: Same as above

File: lib/emergencyNotification.js (hunk 8 above)
Comment (jchavez443) [reply lib/emergencyNotification.js:194]: Same as above

File: lib/aeuis.js (hunk 1 above)
Comment (izakfilmalter) [reply lib/aeuis.js:4]: Don't think classes would be better per say. The sdk uses objects and functions as top level which is confusing. See #11

File: lib/aeuis.js (hunk 2 above)
Comment (izakfilmalter) [reply lib/aeuis.js:29]: Then why did you use the sting template for `/${acid}`? Idk you might as well do the who thing.

**Pull request merged**